src/modules/cloudinary/services/cloudinary.service.ts

```diff
@@ -0,0 +1,42 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { UploadApiErrorResponse, UploadApiResponse, v2 } from 'cloudinary';
+import toStream = require('buffer-to-stream');
```

Comment:
Please, let's do not use commonjs syntax if possible. Better `import * as toStream from 'buffer-to-stream'`